StonePaperScissor - GUI/RockPaperScissors .ipynb

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
.py file is needed and not notebooks

Reply:
> .py file is needed and not notebooks

@kaustubhgupta Exactly .py file is more comfort and i run this program in my local , i didn't find any errors except there are some useless white spaces are exits .